tests/issue/109/unit/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 3.0)
+
+include("../../../cmake/modules/hunter_boost_component_b2_args.cmake")
+project(TestIssue109Unit)
+
+
+set(component "foo")
+set(cmakeArgs "FOO_OPT=1")
+
+hunter_boost_component_b2_args(
+  "bar"
+  ${cmakeArgs}
+  negativeList
+  )
+
+if(negativeList)
+  message(FATAL_ERROR "negativeList should have been empty.")
+endif()
+
+hunter_boost_component_b2_args(
+  "foo"
+  ${cmakeArgs}
+  positiveList
+  )
+
+list(GET positiveList -1 last_item)
+if(NOT last_item STREQUAL "OPT=1")
```

Comment:
May be we can simply check `string(COMPARE EQUAL "${positiveList}" "-s;OPT=1" is_good)`?